submodules/devices/devices.js

```diff
@@ -1071,6 +1083,7 @@ define(function(require) {
 										.concat(
 											type === 'combo_keys' ? ['line'] : []
 										)
+										.filter(isActionTypeAvailableByBrand)
```

Comment:
Using a filtering function isn't warranted as it is only used once. Now that we have access to `keyActionsMod`, we can simply filter out unwanted actions when modding is set:
```suggestion
										.filter(function(action) {
										    return _.isEmpty(keyActionsMod) || _.includes(keyActionsMod, action);
										})
```